test/functional/lua/vim_spec.lua

```diff
@@ -861,4 +861,23 @@ describe('lua stdlib', function()
     eq({}, exec_lua[[return {re1:match_line(0, 1, 1, 7)}]])
     eq({0,3}, exec_lua[[return {re1:match_line(0, 1, 0, 7)}]])
   end)
-end)
+
+  it('vim.defer_fn', function()
+    exec_lua [[
+    vim.g.test = 0
+    vim.defer_fn(function() vim.g.test = 1 end, 5)
+    vim.cmd("sleep 10m")
+    ]]
+    eq(1, exec_lua[[return vim.g.test]])
+  end)
```

Comment:
Could you add not yet executed defered function test?

Reply:
you mean add an `eq(0, exec_lua[[return vim.g.test]])` _before_ the sleep? Sure, I can do that.